buildconf.py

```diff
@@ -117,7 +118,9 @@ def clonePackage(cfg, package, server, gitPackage, branch):
             cmd = ["git", "clone", "-o", "autobuild", "-q", server+gitPackage, clonePath]
             if branch:
                 cmd += ["-b", branch]
-            execute.do(cmd, cfg)
+            out, err, r = execute.do(cmd, cfg)
+            print out
```

Comment:
Do we want this?

Reply:
@AlexanderFabisch it is usefull to have output message for debug purposes. is there any debug mode?

Reply:
I don't think so. @malter should know this.

Reply:
@AlexanderFabisch 
what is the reason not to have output message for debug purpose?

Reply:
At the moment, this would be the only command that actually prints output to stdout. If we added a debug mode, that would be OK but that should be done in another pull request.